profiles/killbill/src/test/java/org/killbill/billing/jaxrs/TestUsage.java

```diff
@@ -64,6 +59,33 @@ public void testRecordUsage() throws Exception {
         addOn.setBillingPeriod(BillingPeriod.NO_BILLING_PERIOD);
         addOn.setPriceList(PriceListSet.DEFAULT_PRICELIST_NAME);
 
+        final Subscriptions subscriptions = new Subscriptions();
+        subscriptions.add(base);
+        subscriptions.add(addOn);
+
+        return subscriptions;
+    }
+
+    private static UUID findSubscriptionIdByProductCategory(final List<Subscription> subscriptions, final ProductCategory category) {
```

Comment:
None of our code is using java fluent apis and instead relies on Guava library. However, we have been talking about upgrading our code base, as tracked by https://github.com/killbill/killbill/issues/1615 so, 👍

Reply:
@sbrossie 

> I understand you needed to touch the pom.xml to add exclusions for your build to run .... 

sorry for this. I already revert pom.xml in `jaxrs` and `profiles-killbill`.

> None of our code is using java fluent apis and instead relies on Guava library.

Hm.. I thought it is safe to use stream, since I saw test in CI using java 11 and I believe that in the docs I see Java 8 used to run killbill.